tests/snuba/api/endpoints/test_organization_events_span_indexed.py

```diff
@@ -525,7 +525,9 @@ def test_query_for_missing_tag(self):
         assert response.data["data"] == [{"foo": "", "count()": 1}]
 
 
-@pytest.mark.xfail(reason="Snuba is prefixing keys, and Sentry wasn't updated first")
+@pytest.mark.xfail(
```

Comment:
not a huge fan of xfailing the test for the actual change you're making, let's just do this in another PR if we break something in snuba again?

Reply:
if we *plan* on breaking that is, now that we know which tests are affected by EAP changes.

Reply:
I disagree, I've had to xfail tests in this file on multiple occasions already. I'm the only person working on this, and these tests don't block Snuba from merging. Which means they only need to pass as I'm working on them locally.i

Reply:
Unless we add these tests as something that needs to pass as part of Snuba's CI I don't see how removing the xfail helps

Reply:
would you xfail them in a separate PR so that we can be confident CI passes for this change at least?

Reply:
aren't there versions of these dests that run against dist tables that you can't run locally?

Reply:
> aren't there versions of these dests that run against dist tables that you can't run locally?

Not that I'm aware of, I've never seen a test pass locally and fail in CI